Dockerfile-msphpsql

```diff
@@ -56,7 +56,7 @@ WORKDIR $PHPSQLDIR/source/
 RUN chmod +x ./packagize.sh
 RUN /bin/bash -c "./packagize.sh"
 
-RUN echo "extension = pdo_sqlsrv.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
+RUN echo "extension = pdo_sqlsrv.so" >> /etc/php/7.0/cli/conf.d/20-pdo_sqlsrv.ini
 RUN echo "extension = sqlsrv.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
 
```

Comment:
does it only work for PHP 7.0?

Reply:
The dockerfile only installs php 7.0 (see lines 20-21) so only 7.0 configuration is included.

Reply:
interesting... perhaps we should at least upgrade to 7.1?

Reply:
Ubuntu 16 defaults to PHP 7.0, so that might require some nontrivial changes to the dockerfile. If we do that, it should be in a separate PR I think.

Reply:
good point
let's not worry about it for now